### DISCUSSION TITLE
fopen from <stdio.h> will return NULL

### DISCUSSION THREAD

**Discussion by linrongbin16** [pybind/pybind11#3582] [Q&A] [answered]
Hi, I'm trying to open a file in pybind11 module and write something into it.

But `FILE *fp = fopen("log/module_cc.log", "w");` will return `NULL`. 

Would you please help me with that ?

Actually I'm trying to use spdlog for logging in pybind11 module, and it seems not work as well.

**Comment by cielavenir** [comment, marked as answer]
have you created "log" dir first?

**Reply by linrongbin16** [reply]
Turns out it's my mistake. I have already fix it.
